src/visual.ts

```diff
@@ -247,69 +145,19 @@ module powerbi.extensibility.visual {
-                percentile: getValue<number>(objects, 'settings_viz_params', 'percentile', 40),
-                weight: getValue<number>(objects, 'settings_viz_params', 'weight', 10),
-            };
-
-            this.settings_labeling_params = <VisualSettingsLabelingParams>{
-                show: getValue<boolean>(objects, 'settings_labeling_params', 'show', true),
-                //addLabel2points: getValue<boolean>(objects, 'settings_labeling_params', 'addLabel2points',false),
-                textSize: getValue<number>(objects, 'settings_labeling_params', 'textSize', 8),
-                percentile: getValue<number>(objects, 'settings_labeling_params', 'percentile', 100),
-                maxLenPointLabel: getValue<number>(objects, 'settings_labeling_params', 'maxLenPointLabel', 5),
-                percentile1: getValue<number>(objects, 'settings_labeling_params', 'percentile1', 100),
-            };
-            this.settings_representative_params = <VisualSettingsRepresentativeParams>{
-                show: getValue<boolean>(objects, 'settings_representative_params', 'show', false),
-                //addLabel2clusterDelegate: getValue<boolean>(objects, 'settings_representative_params', 'addLabel2clusterDelegate', false),  
-                textSize: getValue<number>(objects, 'settings_representative_params', 'textSize', 8),
-                maxLenDelegateLabel: getValue<number>(objects, 'settings_representative_params', 'maxLenDelegateLabel', 30)
-
-            };
-            this.settings_legend_params = <VisualSettingsLegendParams>{
-                show: getValue<boolean>(objects, 'settings_legend_params', 'show', true),
-                addLegend: getValue<boolean>(objects, 'settings_legend_params', 'addLegend', true),
-                palleteType: getValue<string>(objects, 'settings_legend_params', 'palleteType', "qPBI"),
-            };
-
-            this.settings_additional_params = <VisualSettingsAdditionalParams>{
-                //show: getValue<boolean>(objects, 'settings_additional_params', 'show', false),
-                sparsify: getValue<boolean>(objects, 'settings_additional_params', 'sparsify', true),
-                minClusters: getValue<number>(objects, 'settings_additional_params', 'minClusters', 2),
-                maxClusters: getValue<number>(objects, 'settings_additional_params', 'maxClusters', 12),
-                maxIter: getValue<number>(objects, 'settings_additional_params', 'maxIter', 10),
-                nStart: getValue<number>(objects, 'settings_additional_params', 'nStart', 5),
-            };
-            //user edit END: update params
-        }
-        
+        // public enumerateObjectInstances(options: EnumerateVisualObjectInstancesOptions):
+        //     VisualObjectInstance[] | VisualObjectInstanceEnumerationObject {
+        //     return VisualSettings.enumerateObjectInstances(this.settings || VisualSettings.getDefault(), options);
+        // }
```

Comment:
how come this is commented out?

Reply:
replaced by user method (like in previous version)